app/uk/gov/hmrc/transitmovementsrouter/connectors/EISConnector.scala

```diff
@@ -75,7 +77,10 @@ class EISConnectorImpl(
   // Used when setting a stream body -- forces the correct content type (default chooses application/octet-stream)
   implicit private val xmlSourceWriter: BodyWritable[Source[ByteString, _]] = BodyWritable(SourceBody, MimeTypes.XML)
 
-  private val HTTP_DATE_FORMATTER = DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss z", Locale.ENGLISH).withZone(ZoneOffset.UTC)
+  private val HTTP_DATE_FORMATTER = DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss", Locale.ENGLISH).withZone(ZoneOffset.UTC)
```

Comment:
Sorry to leap in out of nowhere but this appeared in my feed 😛 Are you sure you aren't looking for [DateTimeFormatter.RFC_1123_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#RFC_1123_DATE_TIME)? It's the conventional way of formatting the date header, see e.g. [MDN's article](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date)

Reply:
:wave:

Unfortunately, EIS want `UTC`, this would give `GMT` (I did have a look!)

Reply:
Huh, weird. I think that's actually not compliant with the [HTTP spec](https://www.rfc-editor.org/rfc/rfc2616#section-3.3):

> All HTTP date/time stamps MUST be represented in Greenwich Mean Time
   (GMT), without exception. For the purposes of HTTP, GMT is exactly
   equal to UTC (Coordinated Universal Time).

I guess I shouldn't be surprised by that at this point 😆

Reply:
Yup, annoyingly so - just threw it through Ammonite:

```
@ import java.time.format.DateTimeFormatter
import java.time.format.DateTimeFormatter

@ import java.time.OffsetDateTime
import java.time.OffsetDateTime

@ import java.time.ZoneOffset
import java.time.ZoneOffset

@ DateTimeFormatter.RFC_1123_DATE_TIME.format(OffsetDateTime.now(ZoneOffset.UTC))
res3: String = "Thu, 13 Apr 2023 11:07:49 GMT"
```